changelogs/fragments/ansible-galaxy-role-search-rc.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - >-
+  ansible-galaxy search rolename - give a warning instead of non-zero return code when search results are empty.
+  This is similar to the behavior when listing roles, which gives a warning if a role cannot be found and exits with rc 0.
```

Comment:
```suggestion
    ``ansible-galaxy search rolename`` - give a warning instead of non-zero return code when search results are empty.
    This is similar to the behavior when listing roles, which gives a warning if a role cannot be found and exits with a return code of ``0``.
```
(broken YAML indentation)

Reply:
Thanks!